src/core/waf.cc

```diff
@@ -46,6 +46,25 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <errno.h>
+
+namespace {
+/// Mark the context with the applied tx. This can be used to avoid performing the
+/// same logic more than once. ie: tolower() on a particular operation that uses the ac
+/// when a previous tx already lowered the string.
+void mark_tx_applied(ns_waflz::rqst_ctx *a_ctx,
+                waflz_pb::sec_action_t_transformation_type_t const &tx_type)
+{
+        switch(tx_type)
+        {
+        case waflz_pb::sec_action_t_transformation_type_t::
```

Comment:
We always use braces around sections following conditional expressions, so as to prevent any possible [goto fail](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) scenarios.  So although it's a little more verbose we would typically write the `switch` as:

```cpp
        switch(a)
        {
        case b:
        {
                <do something>...
                break;
        }
        ...
        default:
        {
                <do something>...
                break;
        }
        }
```